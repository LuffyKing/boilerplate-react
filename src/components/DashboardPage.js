import React from 'react';

const DashboardPage = () => (
  <div>
    dashboard page contents
  </div>
);
export default DashboardPage;
