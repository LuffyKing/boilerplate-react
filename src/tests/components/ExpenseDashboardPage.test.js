import React from 'react';
import {shallow} from 'enzyme';
import DashboardPage from '../../components/DashboardPage';
test('Should test ExpenseDashboard component',()=>{
  const wrapper = shallow(<DashboardPage></DashboardPage>);
  expect(wrapper).toMatchSnapshot();
});
