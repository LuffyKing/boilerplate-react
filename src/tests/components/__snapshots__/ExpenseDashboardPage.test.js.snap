// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test ExpenseDashboard component 1`] = `
<div>
  dashboard page contents
</div>
`;
